R/core_serialize.R

```diff
@@ -17,8 +17,21 @@
 # jobj -> Object, where jobj is an object created in the backend
 # nolint end
 
+get_type <- function(object, types = NULL) {
```

Comment:
@javierluraschi Could you review this change right here? Basically it's to accommodate things like
```
> int_1 <- forge::cast_scalar_integer(1)
> int_1
[1] 1
> class(int_1)
[1] "forge_stamped" "integer"   
```